bin/setenv-docker-customize.sh

```diff
@@ -376,53 +397,8 @@ else
   add_in_exo_configuration "exo.email.smtp.socketFactory.port="
   add_in_exo_configuration "exo.email.smtp.socketFactory.class="
 
-  # Cluster configuration
-  if [ "${EXO_CLUSTER}" = "true" ]; then
-    add_in_exo_configuration "exo.cluster.node.name=${EXO_CLUSTER_NODE_NAME}"
-    JCR_CLUSTER_HOSTS=""
-    SERVICE_CLUSTER_HOSTS=""
-    COMETD_CLUSTER_HOSTS=""
-
-    for cluster_host in $(echo ${EXO_CLUSTER_HOSTS} | tr ',' ' '); do
-      JCR_CLUSTER_HOSTS="${JCR_CLUSTER_HOSTS}${cluster_host}[${EXO_JGROUPS_JCR_PORT}],"
-      SERVICE_CLUSTER_HOSTS="${SERVICE_CLUSTER_HOSTS}${cluster_host}[${EXO_JGROUPS_SERVICE_PORT}],"
-      COMETD_CLUSTER_HOSTS="${COMETD_CLUSTER_HOSTS}http://${cluster_host}:8080/cometd/cometd,"
-    done
-
-    # JGROUPS properties
-    add_in_exo_configuration "exo.jcr.cluster.jgroups.tcpping.initial_hosts=${JCR_CLUSTER_HOSTS}"
-    add_in_exo_configuration "exo.jcr.cluster.jgroups.tcp.bind_addr=${EXO_JGROUPS_ADDR}"
-    add_in_exo_configuration "exo.jcr.cluster.jgroups.tcp.bind_port=${EXO_JGROUPS_JCR_PORT}"
-
-    add_in_exo_configuration "exo.service.cluster.jgroups.tcpping.initial_hosts=${SERVICE_CLUSTER_HOSTS}"
-    add_in_exo_configuration "exo.service.cluster.jgroups.tcp.bind_addr=${EXO_JGROUPS_ADDR}"
-    add_in_exo_configuration "exo.service.cluster.jgroups.tcp.bind_port=${EXO_JGROUPS_SERVICE_PORT}"
-
-    # WebSocket configuration
-    add_in_exo_configuration "exo.cometd.oort.url=http://${EXO_CLUSTER_NODE_NAME}:8080/cometd/cometd"
-    add_in_exo_configuration "exo.cometd.oort.configType=static"
-    add_in_exo_configuration "exo.cometd.oort.cloud=${COMETD_CLUSTER_HOSTS}"
-
-    # JCR configuration
-    add_in_exo_configuration "gatein.jcr.config.type=cluster"
-    # TODO allow to customize this
-    add_in_exo_configuration "gatein.jcr.index.changefilterclass=org.exoplatform.services.jcr.impl.core.query.ispn.LocalIndexChangesFilter"
-
-  fi
-
   # JMX configuration
   if [ "${EXO_JMX_ENABLED}" = "true" ]; then
-    # insert the listener before the "Global JNDI resources" line
```

Comment:
this properties are in use for us. Please keep them

Reply:
I know that it's used by you and all others (devs, ops, community ...). In fact this configuration is deprecated in Tomcat and will be removed as [mentioned here](https://tomcat.apache.org/tomcat-8.5-doc/config/listeners.html#JMX_Remote_Lifecycle_Listener_-_org.apache.catalina.mbeans.JmxRemoteLifecycleListener)

In addition, in PRD 6.1 we will upgrade to Tomcat 9, so I will revert this until we 'll do the upgrade in product and let you re-work it.

(Note: The new option to use is `-Dcom.sun.management.jmxremote.rmi.port=XYZ`, see https://tomcat.apache.org/tomcat-9.0-doc/monitoring.html)

Reply:
Done